components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -67,12 +68,22 @@ export default function Preferences() {
         [toast, setUser, workspaceTimeout],
     );
 
+    const clearCachedStartWorkspaceOptions = useCallback(async () => {
+        if (!user) {
+            return;
+        }
+        AdditionalUserData.set(user, { workspaceCreationOptionsPerCloneURL: [] });
+        setUser(user);
+        await getGitpodService().server.updateLoggedInUser(user);
+        toast("Your cached start options were cleared.");
```

Comment:
**praise**: Forgot to say, great use of the toast component. Cc @selfcontained because https://github.com/gitpod-io/gitpod/pull/17030